ui/src/app/applications/components/filter/filter.tsx

```diff
@@ -110,8 +116,12 @@ export const Filter = (props: FilterProps) => {
         }
     }, [props.selected.length]);
 
+    const totalCount = options.reduce((countSum, option) => {
+        return countSum + option.count;
+    }, 0);
+
     return (
-        <div className='filter'>
+        <div className='filter' key={totalCount}>
```

Comment:
Will this `{totalCount}` be unique for all filters?

Reply:
They won't be unique - I have appended props.label, does that make sense?